api.md

```diff
@@ -937,138 +1021,101 @@ ALTER TABLE <table_name> SET (timescale.compress, timescale.compress_orderby
 | `column_name` | Name of the column used to order by and/or segment by |
 
 #### Sample Usage [](compression_alter-table-sample)
-Set Hypertable to use ORDER BY option for data organization
+Configure a hypertable that ingests device data to use compression.
 
 ```sql
-ALTER TABLE metrics SET (timescaledb.compress, timescaledb.compress_orderby = 'device_id, time');
+ALTER TABLE metrics SET (timescaledb.compress, timescaledb.compress_orderby = 'time DESC', timescaledb.compress_segmentby = 'device_id');
 ```
 
-Set Hypertable to use ORDER BY and SEGMENT BY options for data organization
-```sql
-ALTER TABLE metrics SET (timescaledb.compress, timescaledb.compress_orderby =
-'time', timescaledb.compress_segmentby = 'device_id');
-```
-
-## Compress Chunk (Compression) :community_function: [](compress_chunk)
-
-compress_chunk function is used to compress a specific chunk.
-
-The syntax is:
-
-``` sql
-SELECT compress_chunk( '<chunk_name>');
-```
-#### Sample Usage [](compress_chunk-sample-usage)
-In this example we will compress chunk 1_2
-``` sql
-SELECT compress_chunk( '_timescaledb_internal._hyper_1_2_chunk');
-```
+## Add Compress Chunk Policy :community_function: [](add_compress_chunk_policy)
+Allows you to set a policy by which the system will compress a chunk
+automatically in the background after it reaches a given age.
 
-#### Required Arguments [](compress_chunk-required-arguments)
+#### Required Arguments [](add_compress_chunk_policy-required-arguments)
 
 |Name|Description|
 |---|---|
-| `chunk_name` | Name of the chunck to be compressed|
-
-## Compress Policy (Compression) :community_function: [](compress_policy)
-Allows you to set a policy  by which the system will compress a chunk
-automatically in the background after it reaches a given age.
+| `table_name` | (REGCLASS) Name of the the policy will act on.|
+| `time_interval` | (INTERVAL or integer) The age after which the policy job will compress chunks.|
 
-The Syntax is:
+The `time_interval` parameter should be specifified differently depending on the type of the time column of the hypertable:
+- For hypertables with TIMESTAMP, TIMESTAMPTZ, and DATE time columns: the time interval should be an INTERVAL type
+- For hypertables with integer-based timestamps: the time interval should be an integer type (this requires
+the [integer_now_func](set_integer_now_func) to be set).
 
-``` sql
-SELECT add_compress_chunks_policy('<table_name>', '<time interval>'::interval);
-```
-#### Sample Usage [](compress_policy-sample-usage)
+#### Sample Usage [](add_compress_chunk_policy-sample-usage)
 In this example we will compress chunks older than 60 days from the 'cpu' table.
```

Comment:
Might as well reword this to match your other changes

Reply:
done.